aws_operators/operators/glue_operators.py

```diff
@@ -0,0 +1,244 @@
+from airflow.models import BaseOperator
+from airflow.exceptions import AirflowException
+from airflow.utils import apply_defaults
+import boto3
+import humps
+import logging
+import time
+
+
+class StartGlueJobRunOperator(BaseOperator):
+
+    ui_color = '#ff9900'
+
+    @apply_defaults
+    def __init__(
+            self,
+            job_name,
+            polling_interval=10,
+            *args,
+            **kwargs
+    ):
+        """
+        Trigger AWS Glue Job function
+
+        :param job_name (string) [REQUIRED] -- the name of the Glue job to start and monitor
+        :param polling_interval (integer) (default: 10) -- time interval, in seconds, to check the status of the job
+        :param job_run_id (string) -- The ID of a previous JobRun to retry.
+        :param arguments (dict) -- The job arguments specifically for this run. For this job run, they replace the default arguments set in the job definition itself.
+        :param timeout (integer) -- The JobRun timeout in minutes.
+        :param max_capacity (float) -- The number of AWS Glue data processing units (DPUs) that can be allocated when this job runs.
+        :param security_configuration (string) -- The name of the SecurityConfiguration structure to be used with this job run.
+        :param notification_property (dict) -- Specifies configuration properties of a job run notification.
+        :param worker_type (string) -- The type of predefined worker that is allocated when a job runs. Accepts a value of Standard, G.1X, or G.2X.
+        :param number_of_workers (integer) -- The number of workers of a defined workerType that are allocated when a job runs.
+
+        Reference: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/glue.html#Glue.Client.start_job_run
+        """
+        super(StartGlueJobRunOperator, self).__init__(*args, **kwargs)
+        self.job_name = job_name
+        self.polling_interval = polling_interval
+        self.__dict__.update(kwargs)
+
+        self.glue_client = boto3.client('glue')
+
+        self.pascalized_args = {humps.pascalize(
+            k): v for k, v in self.__dict__.items()}
+        boto3_glue_arguments = ['JobName', 'JobRunId', 'Arguments', 'Timeout', 'MaxCapacity',
+                                'SecurityConfiguration', 'NotificationProperty', 'WorkerType', 'NumberOfWorkers']
+        self.func_args = {key: self.pascalized_args[key] for key in set(
+            boto3_glue_arguments).intersection(self.pascalized_args.keys())}
+
+    def execute(self, context):
+
+        logging.info(self.func_args)
```

Comment:
Provide a bit more detail in the log to say that these are the functions. Else its just content without context.
Example:
logging.info("start_job_run Arguments: " + self.func_args)